end-user-surveys/otel-prom-interoperability/README.md

```diff
@@ -0,0 +1,18 @@
+# OTel x Prometheus Interoperability Survey
+
+## Purpose
+
+The purpose of this survey is to:
+* Determine user sentiment around Prometheus Metric naming
+* Determine if there is a difference of opinions between user groups:
+    * Prometheus "experts" vs OTel "experts"
+    * Developers (writing instrumentation) vs Operators (running agents, writing queries/alerts)
+    * OSS Users (Collector, Prom Server, Thanos) vs users of Managed Services.
+* Learn overall sentiment about the compatibility of the two projects
+
+## Resources
+
+* [CSV Survey Results](/end-user-surveys/otel-prom-interoperability/otel-prom-interoperability-survey.csv)
```

Comment:
* CSV survey results
* PDF survey results
* Blog summary of survey results
* User feedback discussion panel